### DISCUSSION TITLE
v4.0.0 Gluetun

### DISCUSSION THREAD

**Discussion by qdm12** [qdm12/gluetun#2385] [Polls]
There is an issue tracking v4.0.0 needed breaking changes: https://github.com/qdm12/gluetun/issues/1113

It's mostly maintenance tasks to throw out old retro-compatible code, which would make mostly my life easier going forward. Nothing major, but a few small environment variables might stop working such as `FIREWALL_DEBUG`.

**Comment by frepke** [comment]
For those who want to stick with their current version, simply add `pull_policy: never` to the compose file (only for Docker Compose).

**Comment by jgeorgeson** [comment]
Under standard semver rules, breaking changes are expected when going from 3.x.y to 4.x.y.
